services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()

```

Comment:
Directly assigning `date` to `order.created_at` might not be necessary since `created_at` is typically auto-generated. Ensure that the `date` parameter is a properly formatted datetime object if you intend to override the default behavior.